js/viz/chart.js

```diff
@@ -784,6 +784,10 @@ var dxChart = AdvancedChart.inherit({
             _extend(pane.canvas, panesCanvases[pane.name]);
         });
 
+        (rotated ? horizontalAxes : verticalAxes).forEach((axis) => {
```

Comment:
Use that._valueAxis instead of (rotated ? horizontalAxes : verticalAxes) expression